### DISCUSSION TITLE
[source-chargebee] Add consent custom fields to streams

### DISCUSSION THREAD

**Discussion by SvensonDW** [airbytehq/airbyte#34378] [Connector Ideas and Features]
### Connector Name

source-chargebee

### Connector Version

0.3.0

### What step the error happened?

During the sync

### Relevant information

In the API returns from the Chargebee customer stream, all custom consent fields (cs_xxx) are included. However, this is not available in Airbyte for loading. This will probably be similar to how custom fields (cf_xxx) are processed currently. 
See https://www.chargebee.com/docs/1.0/consent-management.html for Chargebee's details on the matter and how it is displayed in the API. 

### Contribute

- [ ] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
Hello @SvensonDW thanks for requesting this. I moved your issue to a discussion to get upvotes and have a better debate about how implement them. Are you interested in implementing this feature?

**Reply by SvensonDW** [reply]
Hello! Sure, thanks. I am not interested in implementing this myself, since I do not have the sufficient knowledge or time to do so. I am an end user that uses this connector quite often in Airbyte 😄

[a comment by an automated account is not shown]
